Description: Fix -Wmaybe-uninitialized compiler warning.
 In particular, the following warning:
 wmcliphist.c: In function 'main':
 wmcliphist.c:386:11: warning: 'icon_file' may be used uninitialized in this function
 [-Wmaybe-uninitialized]
    surface = cairo_image_surface_create_from_png(icon_file);
            ^
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2014-11-23

--- a/wmcliphist.c
+++ b/wmcliphist.c
@@ -203,6 +203,9 @@
 	/* creat dock icon */
 	dock_app = foo_create_main_icon_window(main_window, icon_size);
 
+	/* set default icon */
+	icon_file = "ico_60x060_gray.png";
+
 	if (icon_size) {
 		/* create icon_mask */
 		if (icon_size == 60) {
